.gitignore

```diff
@@ -1,6 +1,6 @@
 .pytest_cache/
 __pycache__/
-venv/
+.venv/
```

Comment:
Brauchen wir die überhaupt? in der Readme ist von Conda die Rede^^

Reply:
Ja, aber sicher ist sicher. Man kann es ja auch mit venv aufsetzen und dann ist es halt unnötig drin. Macht ja keine Schmerzen, wenn es zu viel ist